packages/wallets/src/features/cfd/modals/SuccessModal/SuccessModal.tsx

```diff
@@ -62,13 +58,17 @@ const SuccessModal: FC<TProps> = ({
                     marketType={marketType}
                     platform={platform}
                     title={
-                        <Localize
-                            i18n_default_text='Your {{platformTitle}}{{demoTitle}} account is ready'
-                            values={{
-                                demoTitle: accountType === localize('demo') ? ` ${accountType}` : '',
-                                platformTitle: PlatformDetails[platform].title,
-                            }}
-                        />
+                        isDemo ? (
+                            <Localize
```

Comment:
@lubega-deriv could we somehow remove duplicated content for mobile/desktop?
maybe create a title variable

Reply:
Updated